tables_en/1-02.csv

```diff
@@ -147,7 +147,7 @@ ng.m-3,nanogram per cubic metre,"SI derived unit of mass concentration; SI base
 1,dimensionless,"Dimensionless; SI base unit equivalent: 1"
 rad.s-1,radian per second,"SI derived unit of angular velocity; SI base unit equivalent: s-1"
 mGal,milligal,"CGS unit of acceleration, equal to one thousandth of a Gal (1 Gal = 0.01 m.s-2)"
-E,Eötvös,"Unit of acceleration divided by distance, 1 E = 1 mGal/10 km; SI base unit equivalent: 1E-10 s-2"
+E,Eötvös,"Unit of acceleration divided by distance, 1 E = 1 mGal/10 km; SI base unit equivalent: 0.01 s-2"

```

Comment:
Needs review, because the revised definition in PDF with comments is this: 

"1 E = 1 Gal/cm; SI base unit equivalent: 1E-9"